yt_dlp/extractor/jwplatform.py

```diff
@@ -73,3 +75,128 @@ def _real_extract(self, url):
+        'info_dict': {
+            'id': 'g9j7Eovo',
+            'title': 'Les Français ne veulent-ils plus travailler ? Retrouvez Le Club Le Figaro Idées',
+            'description': 'md5:862b8813148ba4bf10763a65a69dfe41',
+            'upload_date': '20230216',
+            'timestamp': 1676581615,
+            'duration': 3076,
+            'thumbnail': r're:^https?://[^?#]+\.(?:jpeg|jpg)',
+            'ext': 'mp4',
+        },
+    }, {
+        'url': 'https://video.lefigaro.fr/embed/figaro/video/intelligence-artificielle-faut-il-sen-mefier/',
+        'md5': '0b3f10332b812034b3a3eda1ef877c5f',
+        'info_dict': {
+            'id': 'LeAgybyc',
+            'title': 'Intelligence artificielle : faut-il s’en méfier ?',
+            'description': 'md5:249d136e3e5934a67c8cb704f8abf4d2',
+            'upload_date': '20230124',
+            'timestamp': 1674584477,
+            'duration': 860,
+            'thumbnail': r're:^https?://[^?#]+\.(?:jpeg|jpg)',
+            'ext': 'mp4',
+        },
+    }]
+
+    _WEBPAGE_TESTS = [{
+        'url': 'https://video.lefigaro.fr/figaro/video/suivez-en-direct-le-club-le-figaro-international-avec-philippe-gelie-9/',
+        'md5': '3972ddf2d5f8b98699f191687258e2f9',
+        'info_dict': {
+            'id': 'QChnbPYA',
+            'title': 'Où en est le couple franco-allemand ? Retrouvez Le Club Le Figaro International',
+            'description': 'md5:6f47235b7e7c93b366fd8ebfa10572ac',
+            'upload_date': '20230123',
+            'timestamp': 1674503575,
+            'duration': 3153,
+            'thumbnail': r're:^https?://[^?#]+\.(?:jpeg|jpg)',
+            'age_limit': 0,
+            'ext': 'mp4',
+        },
+    }, {
+        'url': 'https://video.lefigaro.fr/figaro/video/la-philosophe-nathalie-sarthou-lajus-est-linvitee-du-figaro-live/',
+        'md5': '3ac0a0769546ee6be41ab52caea5d9a9',
+        'info_dict': {
+            'id': 'QJzqoNbf',
+            'title': 'La philosophe Nathalie Sarthou-Lajus est l’invitée du Figaro Live',
+            'description': 'md5:c586793bb72e726c83aa257f99a8c8c4',
+            'upload_date': '20230217',
+            'timestamp': 1676661986,
+            'duration': 1558,
+            'thumbnail': r're:^https?://[^?#]+\.(?:jpeg|jpg)',
+            'age_limit': 0,
+            'ext': 'mp4',
+        },
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+
+        player_data = self._search_nextjs_data(webpage, display_id)['props']['pageProps']['pageData']['playerData']
+
+        return self.url_result(
+            f'jwplatform:{player_data["videoId"]}', title=player_data.get('title'),
+            description=player_data.get('description'), thumbnail=player_data.get('poster'))
+
+
+class LeFigaroVideoSectionIE(InfoExtractor):
+    _VALID_URL = r'https?://video\.lefigaro\.fr/figaro/(?P<id>[\w-]+)/?$'
```

Comment:
Never use $ in VALID_URLs. I noticed it in your other PRs too. To block more `/...`, use 
```suggestion
    _VALID_URL = r'https?://video\.lefigaro\.fr/figaro/(?P<id>[\w-]+)/?(?:[#?]|$)'
```